src/components/Onboarding/OnboardingBreadcrumb.js

```diff
@@ -0,0 +1,32 @@
+// @flow
+
+import React from 'react'
+import { connect } from 'react-redux'
+import findIndex from 'lodash/findIndex'
+
+import type { OnboardingState } from 'reducers/onboarding'
+
+import Breadcrumb from 'components/Breadcrumb'
+
+const mapStateToProps = state => ({
+  onboarding: state.onboarding,
```

Comment:
create a selector in `reducers/onboarding` like `onboardingSelector` , and here you can do `createStructuredSelector({ onboarding: onboardingSelector })`

`import { createStructuredSelector } from "reselect"`

we'll try to take this approach a bit more everywhere